Documentation/Diagnostics/PH2137.md

```diff
@@ -0,0 +1,56 @@
+# PH2137: Regex needs a timeout
+
+| Property | Value  |
+|--|--|
+| Package | Not published (yet) |
```

Comment:
I would just give the Package Name.  (Until it's published, no one will find this page anyway.)

Reply:
Agree